Index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="navContainer">
+                <div ><a href="#"><span class="nick">Elio Nel</span></a></div>
+                <div ><img src="./icons/IconMenu.png" alt="menu icon"></div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <div class="sectionContainer">
+                <div class="upperCircle">
+                </div>
+                <div class="lowerCircle">
+                </div>
+                <div class="name">
+                    <h1>Elio Cortés</h1>
+                </div>
+                <div>
+                    <h2>Software Developer</h2>
+                </div>
+
+                <div>
+                    <p class="paragraph">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+                
+            </div>
+        </section>
+    </main>
+    <footer>
+        <div class="socialContent"> <ul>
+            <li><a href="#"><img src="./icons/github.png" alt="github icon"></a></li>
+            <li><a href="#"><img src="./icons/linkedIn.png" alt="linkedIn icon"></a></li>
+            <li><a href="#"><img src="./icons/angelist.png" alt="angelist icon"></a></li>
+            <li><a href="#"><img src="./icons/twitter.png" alt="twitter icon"></a></li>
+            <li><a href="#"><img src="./icons/medium.png" alt="medium icon"></a></li>
+        </ul></div>
+       
+    </footer>
```

Comment:
@Deepakdanger: about this issue, I have to tell that: Although there is no common standard about tags nesting the HTML5, the <main> tag is used as a grouping within the <body>, I kindly do not think it is necessary to include this content inside <section>, because <section> is used to mark the parts of the <main> tag.

Reply:
@NeckerFree, The use of the `section` tag is in the requirement. this `section` includes all intro, skills, languages, frameworks, and icons. As this is part of a big project you cant use the `footer` right now. Hope it will help you 😃